front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -294,12 +299,18 @@ export default function AssistantBuilder({
         );
       }
 
-      setAvatarUrls(
+      setDroidAvatarUrls(
```

Comment:
@fontanierh I did not bother updating the `availableUrls` as there's a todo to improve that but let me know if you this this _should be_ part of this PR.

Reply:
Does it mean that it won't be able to pick a spirit randomly ?

Reply:
I added a commit, now it can 🙇🏻‍♀️